plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/thrift/TestHiveMetastoreMetadataQueriesAccessOperations.java

```diff
@@ -601,30 +601,30 @@ public void testSelectColumnsWithLikeOverColumn(boolean allTablesViewsImplemente
         assertMetastoreInvocations("SELECT * FROM information_schema.columns WHERE column_name LIKE 'n%'",
                 allTablesViewsImplemented
                         ? ImmutableMultiset.builder()
-                        .add(GET_ALL_DATABASES)
-                        .add(GET_ALL_TABLES)
-                        .add(GET_ALL_VIEWS)
-                        .addCopies(GET_TABLE, TEST_ALL_TABLES_COUNT)
-                        .build()
+                                .add(GET_ALL_DATABASES)
+                                .add(GET_ALL_TABLES)
```

Comment:
We need to agree on intellij configuration.
The current ("before") code is like my intellij happens to format the code.
The new ("after") code at least my intellij would not accept, and would revert back to the indentation "before"

Reply:
I believe my formatter was taken from the same repo. Let me update it.

Reply:
I noticed that even without formatter _definition_ changes, different intellij versions may render the code slightly differently. For pragmatic reasons we should for now simply standardize on eg latest stable ide version.

for this PR, feel free to simply undo formatting changes.